### DISCUSSION TITLE
Link support

### DISCUSSION THREAD

**Discussion by Baker0o7** [gokadzev/Musify#161] [Ideas]
Allow link support ,thank you for your hard work sir👍

**Comment by gokadzev** [comment]
> Allow link support ,thank you for your hard work sir+1

If you mean search with link, it's already available, just paste link instead of words in the search field and that's it.

**Comment by Baker0o7** [comment]
Thanks Sir but I meant instead of copying the link and go to paste  on android 12 link can be added in setting pressing the link and take you direct to the app 🙏 that what I meant if its possible
